src/main/java/Week8Homework/ShootingResults.java

```diff
@@ -0,0 +1,6 @@
+package Week8Homework;
+
+public enum ShootingResults {
```

Comment:
```suggestion
public enum ShootingResult {
```
class name should be singular